source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,130 @@
+     - OAuth options
+     - Custom branding
+     - MFA token
+     - Default signup
+     - OAuth options are available
+     - Custom branding is configured
+     - Email verification is enabled
+ - Changed **Actions** post menu hover text to **Message Actions**.
+ - Enabled the new onboarding task list for end users.
+ - The legacy ``Enable post formatting`` setting is now hidden when Advanced Text editor is enabled.
+ 
+#### Performance
+ - Improved the performance of aggregate queries related to Collapsed Reply Threads.
+
+#### Administration
+ - Timestamps are now enabled in the default audit configuration.
+ 
+#### Enterprise Subscription
+ - Upgraded the minor version of the ElasticSearch development Docker image.
+ - The Support Packet now contains two additional fields in the ``support_packet.yaml`` file: Active users and License-supported users.
+
+### Bug Fixes
+ - Fixed an issue with ADA Accessibility where screen readers did not TAB to or read "This channel has guests" in the channel header bar.
+ - Fixed an issue where the at-mention autosuggest of users was no longer grouped by channel membership status.
+ - Fixed an issue where the New Messages toast was not fully tappable in the mobile web view.
+ - Fixed an issue where the shortcut modal for channel info showed ``ALT`` instead of ``SHIFT`` for Mac.
+ - Fixed an issue where the **Help > Report a Problem** link was not hidden when a URL was not set for **System Console > Customization > Report a Problem**.
+ - Fixed an issue with the timing of selector performance metrics.
+ - Fixed an issue where the S3 **Test Connection** button deceptively failed unless the user pressed **Save** first.
+ - Fixed an issue where Workspace Optimization did not load on subpath servers.
+ - Fixed an issue where an error was logged when ``SendEmailNotifications`` was not true.
+   
+### config.json
+Multiple setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to Team Edition and Enterprise Edition:
+ - Under ``ServiceSettings`` in ``config.json``:
+   - Changed ``SessionLengthWebInDays`` to ``SessionLengthWebInHours``.
+   - Changed ``SessionLengthMobileInDays`` to ``SessionLengthMobileInHours``.
+   - Changed ``SessionLengthSSOInDays`` to ``SessionLengthSSOInHours``.
+   - The value of ``TrustedProxyIPHeader`` defaults to empty from now on. A previous bug prevented this from happening in certain conditions. Customers are requested to check for these values in their config and set them to nil if necessary.
+   - The default for ``CollapsedThreads`` has been changed to ``always_on``. This does not affect existing configurations where this value is already set to some other value.
+ - Under ``ExperimentalSettings`` in ``config.json``:
+   - Added a new config setting ``EnableAppBar`` to enable and disable the new App Bar.
```

Comment:
@itao, @streamer45: Do we want to add here a recommendation to enable the App Bar for servers with Calls enabled? I know it's already in the setup docs for Calls, but just in case.

Reply:
+1 as long as we explain the correlation between the two (e.g. dedicated space for start/join button)